src/OrchardCore/OrchardCore.Abstractions/Shell/Removing/IShellRemovingManager.cs

```diff
@@ -0,0 +1,20 @@
+using System.Threading.Tasks;
+
+namespace OrchardCore.Environment.Shell.Removing;
+
+/// <summary>
+/// Host service managing the removing of a given tenant.
```

Comment:
I agree with the removal of a "tenant" happening, not a "shell" :). I suggest using this terminology throughout.

Reply:
Okay, will search where it has to be updated

Reply:
Thanks. Not all cases of "shell" under this PR are affected though, but when we're talking about deleting the persisted data, I think we're deleting a tenant, not a shell.

Reply:
Okay I got it.